ui/app/controllers/clients/client/index.js

```diff
@@ -338,13 +338,4 @@ export default class ClientController extends Controller.extend(
     }
   }
   // #endregion metadata
-
```

Comment:
As a side-effect here, I've decided to always show the "Actions" column on clients/:clientid pages (under alloc table).

Before, this ended up looking a bit jarring and now it'll take up a small amount of space but with the benefit of a consistent look/feel. See also the {{true}} pass in the client index handlebars file.

Reply:
+1 on avoid conditional rendering as it can make the UI surprising (why do I only see this thing sometimes?)

More over though, I think I would not have an `Actions` column at all. `AllocationRow` is already quite crowded and adding more rows just makes everything more "squishy" and knowing the number of actions in an alloc doesn't seem very relevant.

Side note: not sure if this will be fixed in a follow-up PR, but the column rendering seems wrong?
https://github.com/hashicorp/nomad/blob/ff928a804590611111763632388161dc711adf88/ui/app/templates/components/allocation-row.hbs#L140-L142

Reply:
Ahh OK, I saw `actions.length` somewhere and thought this would be a number 😅 

Not sure why I didn't see the `...` button 🤔